pkg/controllers/federate/controller.go

```diff
@@ -423,17 +425,17 @@ func (c *FederateController) handleExistingFederatedObject(
 ) (bool, error) {
 	logger := klog.FromContext(ctx)
 
-	logger.V(6).Info("Checking if federated object needs update")
+	logger.V(2).Info("Checking if federated object needs update")
```

Comment:
At V(2), this line will be printed too many times. Can we bump it to V(3) or remove it?